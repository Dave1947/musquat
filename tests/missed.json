{"google.com":true,"facebook.com":true,"youtube.com":true,"yahoo.com":true,"baidu.com":true,"wikipedia.org":true,"live.com":true,"amazon.com":true,"qq.com":true,"twitter.com":true,"taobao.com":true,"linkedin.com":true,"blogspot.com":true,"google.co.in":true,"yahoo.co.jp":true,"sina.com.cn":true,"google.de":true,"yandex.ru":true,"msn.com":true,"wordpress.com":true,"google.co.jp":true,"ebay.com":true,"bing.com":true,"google.com.hk":true,"google.co.uk":true,"google.fr":true,"vk.com":true,"microsoft.com":true,"weibo.com":true,"babylon.com":true,"163.com":true,"googleusercontent.com":true,"tumblr.com":true,"mail.ru":true,"pinterest.com":true,"apple.com":false,"google.com.br":false,"tmall.com":false,"soso.com":false,"paypal.com":false,"google.es":false,"google.ru":false,"google.it":false,"blogger.com":false,"xhamster.com":false,"adobe.com":false,"fc2.com":false,"craigslist.org":false,"imdb.com":false,"sohu.com":false,"go.com":true,"bbc.co.uk":true,"xvideos.com":true,"hao123.com":true,"ask.com":true,"google.com.mx":true,"conduit.com":true,"youku.com":true,"amazon.co.jp":true,"google.ca":true,"ifeng.com":false,"flickr.com":false,"odnoklassniki.ru":false,"avg.com":false,"t.co":false,"amazon.de":false,"cnn.com":false,"livejasmin.com":false,"pornhub.com":false,"aol.com":false,"rakuten.co.jp":false,"google.co.id":false,"thepiratebay.se":false,"espn.go.com":false,"ebay.de":false,"amazon.co.uk":true,"alibaba.com":true,"mywebsearch.com":true,"tudou.com":true,"huffingtonpost.com":true,"zedo.com":true,"adf.ly":true,"stackoverflow.com":true,"alipay.com":true,"google.com.tr":true,"redtube.com":false,"360buy.com":false,"google.com.au":false,"about.com":false,"sogou.com":false,"instagram.com":false,"ebay.co.uk":false,"livedoor.com":false,"google.pl":false,"dailymotion.com":false,"imgur.com":false,"uol.com.br":false,"netflix.com":false,"360.cn":false,"walmart.com":false,"wordpress.org":true,"adcash.com":true,"cnet.com":true,"bp.blogspot.com":true,"nytimes.com":true,"mediafire.com":true,"youporn.com":true,"ameblo.jp":true,"incredibar.com":true,"godaddy.com":true,"chinaz.com":false,"globo.com":false,"neobux.com":false,"xnxx.com":false,"douban.com":false,"wigetmedia.com":false,"google.nl":false,"4shared.com":false,"doubleclick.com":false,"ehow.com":false,"vimeo.com":false,"kat.ph":false,"livejournal.com":false,"google.com.sa":false,"google.com.ar":false,"renren.com":true,"dailymail.co.uk":true,"bankofamerica.com":true,"searchnu.com":true,"reddit.com":true,"dropbox.com":true,"indiatimes.com":true,"blogfa.com":true,"search-results.com":true,"deviantart.com":true,"tube8.com":false,"goo.ne.jp":false,"google.com.eg":false,"nicovideo.jp":false,"blogspot.com.br":false,"foxnews.com":false,"weather.com":false,"badoo.com":false,"bestbuy.com":false,"chase.com":false,"cnzz.com":false,"pconline.com.cn":false,"amazonaws.com":false,"etsy.com":false,"tianya.cn":false,"secureserver.net":true,"spiegel.de":true,"google.cn":true,"google.com.pk":true,"sweetim.com":true,"aweber.com":true,"google.co.th":true,"amazon.cn":true,"google.co.za":true,"stumbleupon.com":true,"people.com.cn":false,"56.com":false,"slideshare.net":false,"sourceforge.net":false,"mozilla.org":false,"warriorforum.com":false,"xinhuanet.com":false,"addthis.com":false,"wikimedia.org":false,"wikia.com":false,"comcast.net":false,"target.com":false,"liveinternet.ru":false,"torrentz.eu":false,"google.co.ve":false,"wellsfargo.com":true,"allegro.pl":true,"outbrain.com":true,"clicksor.com":true,"photobucket.com":true,"depositfiles.com":true,"letv.com":true,"answers.com":true,"softonic.com":true,"squidoo.com":true,"themeforest.net":false,"skype.com":false,"google.com.my":false,"fiverr.com":false,"fbcdn.net":false,"zol.com.cn":false,"uploaded.net":false,"salesforce.com":false,"onet.pl":false,"reference.com":false,"soundcloud.com":false,"rapidshare.com":false,"guardian.co.uk":false,"ucoz.ru":false,"google.gr":false,"google.com.vn":true,"optmd.com":true,"hootsuite.com":true,"google.be":true,"booking.com":true,"download.com":true,"google.com.co":true,"etao.com":true,"pengyou.com":true,"aliexpress.com":true,"statcounter.com":false,"bild.de":false,"rediff.com":false,"scribd.com":false,"files.wordpress.com":false,"free.fr":false,"5000pk.com":false,"flipkart.com":false,"myspace.com":false,"yesky.com":false,"kakaku.com":false,"archive.org":false,"leboncoin.fr":false,"blogspot.com.es":false,"isohunt.com":false,"csdn.net":true,"samsung.com":true,"w3schools.com":true,"amazon.fr":true,"it168.com":true,"58.com":true,"yieldmanager.com":true,"ups.com":true,"profitclicking.com":true,"google.com.ua":true,"media.tumblr.com":false,"domaintools.com":false,"espncricinfo.com":false,"filestube.com":false,"google.com.tw":false,"youjizz.com":false,"wsj.com":false,"telegraph.co.uk":false,"yelp.com":false,"tlbb8.com":false,"xunlei.com":false,"web.de":false,"rutracker.org":false,"google.at":false,"github.com":false,"hatena.ne.jp":true,"2345.com":true,"taringa.net":true,"libero.it":true,"google.ro":true,"nbcnews.com":true,"google.se":true,"ilivid.com":true,"orange.fr":true,"siteadvisor.com":true,"rambler.ru":false,"wp.pl":false,"soku.com":false,"paipai.com":false,"tlbb2.com":false,"sergey-mavrodi.com":false,"hp.com":false,"cam4.com":false,"gmx.net":false,"google.com.ph":false,"dell.com":false,"naver.com":false,"indeed.com":false,"pandora.com":false,"china.com":false,"aili.com":true,"putlocker.com":true,"hostgator.com":true,"xing.com":true,"mercadolivre.com.br":true,"washingtonpost.com":true,"ero-advertising.com":true,"google.com.ng":true,"popads.net":true,"wordreference.com":true,"hardsextube.com":false,"hudong.com":false,"imageshack.us":false,"php.net":false,"thefreedictionary.com":false,"google.com.pe":false,"ikea.com":false,"kaixin001.com":false,"google.ch":false,"ameba.jp":false,"adultfriendfinder.com":false,"torrentz.in":false,"forbes.com":false,"4399.com":false,"tlbbsifu.com":false,"tripadvisor.com":true,"mgid.com":true,"google.com.sg":true,"tagged.com":true,"youdao.com":true,"movie2k.to":true,"in.com":true,"hulu.com":true,"reuters.com":true,"narod.ru":true,"weebly.com":false,"dangdang.com":false,"pof.com":false,"dianxin.cn":false,"extratorrent.com":false,"newegg.com":false,"360tl.com":false,"google.cl":false,"googleadservices.com":false,"nfl.com":false,"bet365.com":false,"linkbucks.com":false,"nba.com":false,"yourlust.com":false,"iqiyi.com":false,"mashable.com":true,"rapidgator.net":true,"groupon.com":true,"twoo.com":true,"usps.com":true,"9gag.com":true,"naver.jp":true,"digitalpoint.com":true,"constantcontact.com":true,"shutterstock.com":true,"pchome.net":false,"goal.com":false,"hurriyet.com.tr":false,"drtuber.com":false,"mixi.jp":false,"americanexpress.com":false,"att.com":false,"ign.com":false,"bitauto.com":false,"fedex.com":false,"kooora.com":false,"cntv.cn":false,"terra.com.br":false,"zanox.com":false,"marca.com":false,"xcar.com.cn":true,"mycalendarbook.com":true,"kohls.com":true,"nih.gov":true,"4dsply.com":true,"ebay.com.au":true,"adnxs.com":true,"milliyet.com.tr":true,"ask.fm":true,"clickbank.com":true,"macys.com":false,"businessinsider.com":false,"funmoods.com":false,"blogspot.de":false,"t-online.de":false,"soufun.com":false,"seesaa.net":false,"ebay.it":false,"alimama.com":false,"ku6.com":false,"huanqiu.com":false,"sears.com":false,"homeway.com.cn":false,"gazeta.pl":false,"goodreads.com":false,"detik.com":true,"joomla.org":true,"ezinearticles.com":true,"vancl.com":true,"2ch.net":true,"xe.com":true,"dmm.co.jp":true,"51job.com":true,"gsmarena.com":true,"seznam.cz":true,"livedoor.biz":false,"google.co.hu":false,"autohome.com.cn":false,"typepad.com":false,"ebay.in":false,"chip.de":false,"dianping.com":false,"uimserv.net":false,"informer.com":false,"zimbio.com":false,"wix.com":false,"drudgereport.com":false,"google.pt":false,"aizhan.com":false,"seomoz.org":false,"elmundo.es":true,"elpais.com":true,"agoda.com":true,"ig.com.br":true,"engadget.com":true,"mailchimp.com":true,"match.com":true,"turbobit.net":true,"kaskus.co.id":true,"blogspot.jp":true,"yandex.ua":false,"nifty.com":false,"ganji.com":false,"cj.com":false,"google.cz":false,"searchfunmoods.com":false,"infusionsoft.com":false,"webs.com":false,"toysrus.com":false,"126.com":false,"speedtest.net":false,"1channel.ch":false,"pch.com":false,"adultadworld.com":false,"iminent.com":false,"ebay.fr":true,"google.dk":true,"ning.com":true,"usatoday.com":true,"bluehost.com":true,"ndtv.com":true,"freelancer.com":true,"leo.org":true,"techcrunch.com":true,"histats.com":true,"verizonwireless.com":false,"irctc.co.in":false,"gutefrage.net":false,"over-blog.com":false,"avito.ru":false,"hdfcbank.com":false,"abcnews.go.com":false,"w3.org":false,"allrecipes.com":false,"google.ae":false,"varzesh3.com":false,"homedepot.com":false,"kuxun.cn":false,"enet.com.cn":false,"surveymonkey.com":false,"vnexpress.net":true,"google.no":true,"beeg.com":true,"nuvid.com":true,"doorblog.jp":true,"letitbit.net":true,"examiner.com":true,"clixsense.com":true,"chinanews.com":true,"mihanblog.com":true,"google.ie":false,"blogspot.ru":false,"battle.net":false,"jimdo.com":false,"bannersbroker.com":false,"retailmenot.com":false,"akamaihd.net":false,"abril.com.br":false,"twitpic.com":false,"myfreecams.com":false,"wikihow.com":false,"sape.ru":false,"commentcamarche.net":false,"lemonde.fr":false,"largeporntube.com":false,"capitalone.com":false,"51buy.com":true,"mop.com":true,"hubpages.com":false,"gap.com":true,"bloomberg.com":false,"mpnrs.com":true,"softpedia.com":false,"pcauto.com.cn":false,"naukri.com":false,"xda-developers.com":false,"10086.cn":false,"google.fi":false,"istockphoto.com":false,"angege.com":false,"youm7.com":false,"bleacherreport.com":false,"jobrapido.com":false,"swagbucks.com":false,"repubblica.it":false,"blogspot.ca":false,"clkads.com":false,"blogspot.it":false,"webmd.com":false,"stackexchange.com":false}
