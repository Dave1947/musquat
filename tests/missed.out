{ '0': 'YES',
  '1': 'YES',
  '2': 'YES',
  '3': 'YES',
  '4': 'YES',
  '5': 'YES',
  '6': 'YES',
  '7': 'YES',
  '8': 'YES',
  '9': 'YES',
  '10': 'YES',
  '11': 'YES',
  '12': 'YES',
  '13': 'YES',
  '14': 'YES',
  '15': 'YES',
  '16': 'YES',
  '17': 'YES',
  '18': 'YES',
  '19': 'YES',
  '20': 'YES',
  '21': 'YES',
  '22': 'YES',
  '23': 'YES',
  '24': 'YES',
  '25': 'YES',
  '26': 'YES',
  '27': 'YES',
  '28': 'YES',
  '29': 'YES',
  '30': 'YES',
  '31': 'YES',
  '32': 'YES',
  '33': 'YES',
  '34': 'YES',
  '35': 'NO',
  '36': 'NO',
  '37': 'NO',
  '38': 'NO',
  '39': 'NO',
  '40': 'NO',
  '41': 'NO',
  '42': 'NO',
  '43': 'NO',
  '44': 'NO',
  '45': 'NO',
  '46': 'NO',
  '47': 'NO',
  '48': 'NO',
  '49': 'NO',
  '50': 'YES',
  '51': 'YES',
  '52': 'YES',
  '53': 'YES',
  '54': 'YES',
  '55': 'YES',
  '56': 'YES',
  '57': 'YES',
  '58': 'YES',
  '59': 'YES',
  '60': 'NO',
  '61': 'NO',
  '62': 'NO',
  '63': 'NO',
  '64': 'NO',
  '65': 'NO',
  '66': 'NO',
  '67': 'NO',
  '68': 'NO',
  '69': 'NO',
  '70': 'NO',
  '71': 'NO',
  '72': 'NO',
  '73': 'NO',
  '74': 'NO',
  '75': 'YES',
  '76': 'YES',
  '77': 'YES',
  '78': 'YES',
  '79': 'YES',
  '80': 'YES',
  '81': 'YES',
  '82': 'YES',
  '83': 'YES',
  '84': 'YES',
  '85': 'NO',
  '86': 'NO',
  '87': 'NO',
  '88': 'NO',
  '89': 'NO',
  '90': 'NO',
  '91': 'NO',
  '92': 'NO',
  '93': 'NO',
  '94': 'NO',
  '95': 'NO',
  '96': 'NO',
  '97': 'NO',
  '98': 'NO',
  '99': 'NO',
  '100': 'YES',
  '101': 'YES',
  '102': 'YES',
  '103': 'YES',
  '104': 'YES',
  '105': 'YES',
  '106': 'YES',
  '107': 'YES',
  '108': 'YES',
  '109': 'YES',
  '110': 'NO',
  '111': 'NO',
  '112': 'NO',
  '113': 'NO',
  '114': 'NO',
  '115': 'NO',
  '116': 'NO',
  '117': 'NO',
  '118': 'NO',
  '119': 'NO',
  '120': 'NO',
  '121': 'NO',
  '122': 'NO',
  '123': 'NO',
  '124': 'NO',
  '125': 'YES',
  '126': 'YES',
  '127': 'YES',
  '128': 'YES',
  '129': 'YES',
  '130': 'YES',
  '131': 'YES',
  '132': 'YES',
  '133': 'YES',
  '134': 'YES',
  '135': 'NO',
  '136': 'NO',
  '137': 'NO',
  '138': 'NO',
  '139': 'NO',
  '140': 'NO',
  '141': 'NO',
  '142': 'NO',
  '143': 'NO',
  '144': 'NO',
  '145': 'NO',
  '146': 'NO',
  '147': 'NO',
  '148': 'NO',
  '149': 'NO',
  '150': 'YES',
  '151': 'YES',
  '152': 'YES',
  '153': 'YES',
  '154': 'YES',
  '155': 'YES',
  '156': 'YES',
  '157': 'YES',
  '158': 'YES',
  '159': 'YES',
  '160': 'NO',
  '161': 'NO',
  '162': 'NO',
  '163': 'NO',
  '164': 'NO',
  '165': 'NO',
  '166': 'NO',
  '167': 'NO',
  '168': 'NO',
  '169': 'NO',
  '170': 'NO',
  '171': 'NO',
  '172': 'NO',
  '173': 'NO',
  '174': 'NO',
  '175': 'YES',
  '176': 'YES',
  '177': 'YES',
  '178': 'YES',
  '179': 'YES',
  '180': 'YES',
  '181': 'YES',
  '182': 'YES',
  '183': 'YES',
  '184': 'YES',
  '185': 'NO',
  '186': 'NO',
  '187': 'NO',
  '188': 'NO',
  '189': 'NO',
  '190': 'NO',
  '191': 'NO',
  '192': 'NO',
  '193': 'NO',
  '194': 'NO',
  '195': 'NO',
  '196': 'NO',
  '197': 'NO',
  '198': 'NO',
  '199': 'NO',
  '200': 'YES',
  '201': 'YES',
  '202': 'YES',
  '203': 'YES',
  '204': 'YES',
  '205': 'YES',
  '206': 'YES',
  '207': 'YES',
  '208': 'YES',
  '209': 'YES',
  '210': 'NO',
  '211': 'NO',
  '212': 'NO',
  '213': 'NO',
  '214': 'NO',
  '215': 'NO',
  '216': 'NO',
  '217': 'NO',
  '218': 'NO',
  '219': 'NO',
  '220': 'NO',
  '221': 'NO',
  '222': 'NO',
  '223': 'NO',
  '224': 'NO',
  '225': 'YES',
  '226': 'YES',
  '227': 'YES',
  '228': 'YES',
  '229': 'YES',
  '230': 'YES',
  '231': 'YES',
  '232': 'YES',
  '233': 'YES',
  '234': 'YES',
  '235': 'NO',
  '236': 'NO',
  '237': 'NO',
  '238': 'NO',
  '239': 'NO',
  '240': 'NO',
  '241': 'NO',
  '242': 'NO',
  '243': 'NO',
  '244': 'NO',
  '245': 'NO',
  '246': 'NO',
  '247': 'NO',
  '248': 'NO',
  '249': 'NO',
  '250': 'YES',
  '251': 'YES',
  '252': 'YES',
  '253': 'YES',
  '254': 'YES',
  '255': 'YES',
  '256': 'YES',
  '257': 'YES',
  '258': 'YES',
  '259': 'YES',
  '260': 'NO',
  '261': 'NO',
  '262': 'NO',
  '263': 'NO',
  '264': 'NO',
  '265': 'NO',
  '266': 'NO',
  '267': 'NO',
  '268': 'NO',
  '269': 'NO',
  '270': 'NO',
  '271': 'NO',
  '272': 'NO',
  '273': 'NO',
  '274': 'NO',
  '275': 'YES',
  '276': 'YES',
  '277': 'YES',
  '278': 'YES',
  '279': 'YES',
  '280': 'YES',
  '281': 'YES',
  '282': 'YES',
  '283': 'YES',
  '284': 'YES',
  '285': 'NO',
  '286': 'NO',
  '287': 'NO',
  '288': 'NO',
  '289': 'NO',
  '290': 'NO',
  '291': 'NO',
  '292': 'NO',
  '293': 'NO',
  '294': 'NO',
  '295': 'NO',
  '296': 'NO',
  '297': 'NO',
  '298': 'NO',
  '299': 'NO',
  '300': 'YES',
  '301': 'YES',
  '302': 'YES',
  '303': 'YES',
  '304': 'YES',
  '305': 'YES',
  '306': 'YES',
  '307': 'YES',
  '308': 'YES',
  '309': 'YES',
  '310': 'NO',
  '311': 'NO',
  '312': 'NO',
  '313': 'NO',
  '314': 'NO',
  '315': 'NO',
  '316': 'NO',
  '317': 'NO',
  '318': 'NO',
  '319': 'NO',
  '320': 'NO',
  '321': 'NO',
  '322': 'NO',
  '323': 'NO',
  '324': 'NO',
  '325': 'YES',
  '326': 'YES',
  '327': 'YES',
  '328': 'YES',
  '329': 'YES',
  '330': 'YES',
  '331': 'YES',
  '332': 'YES',
  '333': 'YES',
  '334': 'YES',
  '335': 'NO',
  '336': 'NO',
  '337': 'NO',
  '338': 'NO',
  '339': 'NO',
  '340': 'NO',
  '341': 'NO',
  '342': 'NO',
  '343': 'NO',
  '344': 'NO',
  '345': 'NO',
  '346': 'NO',
  '347': 'NO',
  '348': 'NO',
  '349': 'NO',
  '350': 'YES',
  '351': 'YES',
  '352': 'YES',
  '353': 'YES',
  '354': 'YES',
  '355': 'YES',
  '356': 'YES',
  '357': 'YES',
  '358': 'YES',
  '359': 'YES',
  '360': 'NO',
  '361': 'NO',
  '362': 'NO',
  '363': 'NO',
  '364': 'NO',
  '365': 'NO',
  '366': 'NO',
  '367': 'NO',
  '368': 'NO',
  '369': 'NO',
  '370': 'NO',
  '371': 'NO',
  '372': 'NO',
  '373': 'NO',
  '374': 'NO',
  '375': 'YES',
  '376': 'YES',
  '377': 'YES',
  '378': 'YES',
  '379': 'YES',
  '380': 'YES',
  '381': 'YES',
  '382': 'YES',
  '383': 'YES',
  '384': 'YES',
  '385': 'NO',
  '386': 'NO',
  '387': 'NO',
  '388': 'NO',
  '389': 'NO',
  '390': 'NO',
  '391': 'NO',
  '392': 'NO',
  '393': 'NO',
  '394': 'NO',
  '395': 'NO',
  '396': 'NO',
  '397': 'NO',
  '398': 'NO',
  '399': 'NO',
  '400': 'YES',
  '401': 'YES',
  '402': 'YES',
  '403': 'YES',
  '404': 'YES',
  '405': 'YES',
  '406': 'YES',
  '407': 'YES',
  '408': 'YES',
  '409': 'YES',
  '410': 'NO',
  '411': 'NO',
  '412': 'NO',
  '413': 'NO',
  '414': 'NO',
  '415': 'NO',
  '416': 'NO',
  '417': 'NO',
  '418': 'NO',
  '419': 'NO',
  '420': 'NO',
  '421': 'NO',
  '422': 'NO',
  '423': 'NO',
  '424': 'NO',
  '425': 'YES',
  '426': 'YES',
  '427': 'YES',
  '428': 'YES',
  '429': 'YES',
  '430': 'YES',
  '431': 'YES',
  '432': 'YES',
  '433': 'YES',
  '434': 'YES',
  '435': 'NO',
  '436': 'NO',
  '437': 'NO',
  '438': 'NO',
  '439': 'NO',
  '440': 'NO',
  '441': 'NO',
  '442': 'NO',
  '443': 'NO',
  '444': 'NO',
  '445': 'NO',
  '446': 'NO',
  '447': 'NO',
  '448': 'NO',
  '449': 'NO',
  '450': 'YES',
  '451': 'YES',
  '452': 'YES',
  '453': 'YES',
  '454': 'YES',
  '455': 'YES',
  '456': 'YES',
  '457': 'YES',
  '458': 'YES',
  '459': 'YES',
  '460': 'NO',
  '461': 'NO',
  '462': 'NO',
  '463': 'NO',
  '464': 'NO',
  '465': 'NO',
  '466': 'NO',
  '467': 'NO',
  '468': 'NO',
  '469': 'NO',
  '470': 'NO',
  '471': 'NO',
  '472': 'NO',
  '473': 'NO',
  '474': 'NO',
  '475': 'NO',
  '476': 'YES',
  '477': 'YES',
  '478': 'NO',
  '479': 'YES',
  '480': 'NO',
  '481': 'YES',
  '482': 'NO',
  '483': 'NO',
  '484': 'NO',
  '485': 'NO',
  '486': 'NO',
  '487': 'NO',
  '488': 'NO',
  '489': 'NO',
  '490': 'NO',
  '491': 'NO',
  '492': 'NO',
  '493': 'NO',
  '494': 'NO',
  '495': 'NO',
  '496': 'NO',
  '497': 'NO',
  '498': 'NO',
  '499': 'NO' }
